lib/download-job.js

```diff
@@ -10,6 +10,7 @@ export async function run(jobRecord) {
 
   let sparqlParams = jobParams.query;
   let sparqlQuery = Queries.Reports.build(sparqlParams);
+  console.log('Queried CSV: ' + sparqlQuery);
```

Comment:
Is dit nog een leftover van development/debugging? Of wel degelijk de bedoeling dat dit gelogd wordt?